test/v2/controllers/V2MovementsControllerSpec.scala

```diff
@@ -3360,16 +3361,15 @@ class V2MovementsControllerSpec
             )
           )
             .thenAnswer(
-              _ => EitherT.leftT(PersistenceError.MovementNotFound(movementId, movementType))
+              _ => EitherT.rightT(List.empty[MessageSummary])
             )
 
           val request = FakeRequest("GET", "/", FakeHeaders(), Source.empty[ByteString])
           val result  = sut.getMessageIds(movementType, movementId, None)(request)
 
-          status(result) mustBe NOT_FOUND
-          contentAsJson(result) mustBe Json.obj(
-            "code"    -> "NOT_FOUND",
-            "message" -> s"${movementType.movementType.capitalize} movement with ID ${movementId.value} was not found"
+          status(result) mustBe OK
+          contentAsJson(result) mustBe Json.toJson(
+            HateoasMovementMessageIdsResponse(movementId, List.empty[MessageSummary], None, movementType)
```

Comment:
When no movement is found for the list messages endpoint, this should still return `NOT_FOUND` -- because the _movement_ was not found,